keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,337 @@
+        .credential(new AzureCredential())
+        .build();
+        
+secretAsyncClient.getSecret("secretName").subscribe(secretResponse ->
+  System.out.printf("Secret with name %s , value %s \n", secretResponse.value().name(),
+  secretResponse.value().value()));
+```
+
+### Update an existing Secret Asynchronously
+
+Update an existing Secret by calling `updateSecret`.
+```Java
+SecretAsyncClient secretAsyncClient = SecretAsyncClient.builder()
+        .endpoint(<your-vault-url>)
+        .credential(new AzureCredential())
+        .build();
+
+secretAsyncClient.getSecret("secretName").subscribe(secretResponse -> {
+     // Get the Secret
+     Secret secret = secretResponse.value();
+     // Update the expiry time of the secret.
+     secret.expires(OffsetDateTime.now().plusDays(50));
+     secretAsyncClient.updateSecret(secret).subscribe(secretResponse ->
+         System.out.printf("Secret's updated not before time %s \n", secretResponse.value().notBefore().toString()));
+   });
+```
+
+### Delete a Secret Asynchronously
+
+Delete an existing Secret by calling `deleteSecret`.
+```Java
+SecretAsyncClient secretAsyncClient = SecretAsyncClient.builder()
+        .endpoint(<your-vault-url>)
+        .credential(new AzureCredential())
+        .build();
+        
+secretAsyncClient.deleteSecret("secretName").subscribe(deletedSecretResponse ->
+   System.out.printf("Deleted Secret's deletion time %s \n", deletedSecretResponse.value().deletedDate().toString()));
+```
+
+### List Secrets Asynchronously
+
+List the secrets in the key vault by calling `listSecrets`.
+```Java
+SecretAsyncClient secretAsyncClient = SecretAsyncClient.builder()
+        .endpoint(<your-vault-url>)
+        .credential(new AzureCredential())
+        .build();
+       
+// The List Secrets operation returns secrets without their value, so for each secret returned we call `getSecret` to get its // value as well.
+secretAsyncClient.listSecrets()
+  .flatMap(secretAsyncClient::getSecret).subscribe(secretResponse ->
+    System.out.printf("Secret with name %s , value %s \n", secretResponse.value().name(), secretResponse.value().value()));
+```
+
+## Troubleshooting
+### General
+Key Vault clients raise exceptions defined in azure-core. For more detailed infromation about exceptions and how to deal with them, see [Azure Core exceptions][azure_core_exceptions].
+
+For example, if you try to retrieve a secret after it is deleted a `404` error is returned, indicating resource not found. In the following snippet, the error is handled gracefully by catching the exception and displaying additional information about the error.
+```java
+try {
+    SecretClient.getSecret("deletedSecret")
+} catch (ResourceNotFoundException e) {
+    System.out.println(e.getMessage());
+}
+```
+### Logging [TODO]
+This SDK uses SLF4J logging library. The logging is enabled by default, just ensure your application/project takes dependency on SLF4J logging library.
```

Comment:
I would suggest you file an issue and remove this for now.

Reply:
Removed,
Filed an issue.
#4016